symfit/core/minimizers.py

```diff
@@ -446,15 +456,87 @@ def execute(self, **minimize_options):
             **minimize_options
         )
 
+    @classmethod
+    def minimize_method(cls):
+        return "L-BFGS-B"
 
 class NelderMead(ScipyMinimize, BaseMinimizer):
     """
     Wrapper around :func:`scipy.optimize.minimize`'s NelderMead algorithm.
     """
+    @classmethod
+    def minimize_method(cls):
+        return 'Nelder-Mead'
+
     def execute(self, **minimize_options):
         return super(NelderMead, self).execute(method='Nelder-Mead', **minimize_options)
 
 
+class BasinHopping(ScipyMinimize, BaseMinimizer):
+    """
+    Wrapper around :func:`scipy.optimize.basinhopping`'s basin-hopping algorithm.
+
+    As always, the best way to use this algorithm is through
+    :class:`~symfit.core.fit.Fit`, as this will automatically select a local
+    minimizer for you depending on wether you provvided ounds, constraints, etc.
```

Comment:
Needless bit of docstring

Reply:
I kinda disagree, because I want to make sure people find enough examples of this syntax in the docs. But you are right, this docstring does not even explain how to use it directly which would make more sense.

Reply:
I'd argue that the docstring should describe on how to use this function/class. The actual docs should use the *best* way of using it.